menus/tree-view.cson

```diff
@@ -39,6 +39,7 @@
     {'label': 'Split Right', 'command': 'tree-view:open-selected-entry-right'}
     {'type': 'separator'}
 
+    {'label': 'Add Root Folder', 'command': 'tree-view:add-root-folder'}
     {'label': 'Copy Full Path', 'command': 'tree-view:copy-full-path'}
```

Comment:
@atom/core Does anyone have an opinion about the naming of this command and the *renaming* of the two above? We thought `Add Folder` seemed to similar to `Add Root Folder`, so we renamed it to `New Folder`, and did the same for `File`.

I thought about renaming the actual commands too, but I held off because I didn't want to break people's key-bindings.